eng/pipelines/templates/steps/ref-updater.yml

```diff
@@ -6,10 +6,10 @@ parameters:
   Repos: []
 
 steps:
-
+- template: /eng/common/pipelines/templates/steps/set-default-branch.yml
```

Comment:
@weshaggard do you know if we are using the ref updater anymore now that we have eng/common?

Reply:
It is used in few places and I'd like to keep the system around it in case we ever need to use for other places. We also do use it for our internal build tools whenever we release a new version of that as well.

Reply:
Fair enough. Just checking in case there was an opportunity to nuke some code.